### PR TITLE
v3.4.0 – Adding more-vertical icon

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- `more-vertical` icon added.

### Changed
- Minor package updates

### CODE REVIEW

**Review by FayeCarter — APPROVED** [review]

**Review by zlatinivanov-je — APPROVED** [review]
Is there a screenshot preview?

File: icons/more-vertical.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+    <path d="M12 8a2 2 0 100-4 2 2 0 000 4zm0 2a2 2 0 100 4 2 2 0 000-4zm0 6a2 2 0 100 4 2 2 0 000-4z" fill="#000" fill-rule="evenodd"/>
```
Comment (kevinrodrigues) [inline icons/more-vertical.svg:2]: Should we be removing the fill prop here so the consumers can set them?

Comment (ashleynolan) [reply icons/more-vertical.svg:2]: This just acts as a default fill. Consumers can override this in CSS by using `fill: xxxx;`

Happy to remove though if we'd rather them just be empty (need to do this on some others too, so happy to do this as a separate PR).

Comment (kevinrodrigues) [reply icons/more-vertical.svg:2]: No that's ok i just notice some are empty and some are there. Happy as is 👍

**Pull request merged**